### ISSUE TITLE
Can you provide the PCBs for it?

### ISSUE THREAD

**Issue by astrolemonade** [clockworkpi/uConsole#22]
Hey! Would love to hack it but there are no PCBs for it. Can you provide the PCB files?
Thank you!

**Comment by matheusmoreira** [comment]
I'd also like to see the PCB files released under copyleft license. Plenty of folks are looking to build custom computr modules for this device, even custom boards. I'd like to learn hardware and electronics by giving it a try as well.

**Comment by shymega** [comment]
Duplicate of #7.

**Comment by DOSputin** [comment]
Yes.  And while you are at it, also publish the CAD files for the various machined parts.

**Comment by shymega** [comment]
I can't do it.
